etl3/app.py

```diff
@@ -0,0 +1,65 @@
+import time
+import config
+import pandas as pd
+from sqlalchemy import create_engine
+
+config = config.get_config()
+
+etl3_db_engine = create_engine(config["etl3_db"]["connection_string"])
+sirius_db_engine = create_engine(config["sirius_db"]["connection_string"])
+etl3_db_schema = config["etl3_db"]["schema_name"]
+
+if __name__ == "__main__":
+    t = time.process_time()
+
+    print("Fetching Sirius IDs...")
+
+    print("- Persons (Clients)")
+    sirius_persons = pd.read_sql_table("persons", con=sirius_db_engine)
+    sirius_persons_keys = sirius_persons[(sirius_persons.type == 'actor_client')][['caserecnumber', 'id']]
+    sirius_persons_keys.columns = ['caserecnumber', 'sirius_persons_id']
+    sirius_persons_keys.to_sql('sirius_map_persons', con=etl3_db_engine, schema=etl3_db_schema, if_exists="replace", index=False)
+
+    print("- Cases")
+    sirius_cases = pd.read_sql_table("cases", con=sirius_db_engine)
+    sirius_cases_keys = sirius_cases[(sirius_cases.casetype == 'ORDER')][['id', 'caserecnumber', 'client_id', 'casesubtype', 'orderdate']]
+    sirius_cases_keys.columns = ['sirius_cases_id', 'sirius_cases_caserecnumber', 'sirius_persons_id', 'sirius_casesubtype', 'orderdate']
+    sirius_cases_keys.to_sql('sirius_map_cases', con=etl3_db_engine, schema=etl3_db_schema, if_exists="replace", index=False)
+
+    print("- Addresses")
+    sirius_addresses = pd.read_sql_table("addresses", con=sirius_db_engine)
+    sirius_addresses_keys = sirius_addresses[['id', 'person_id']]
+    sirius_addresses_keys.columns = ['sirius_addresses_id', 'sirius_persons_id']
+    sirius_addresses_keys.to_sql('sirius_map_addresses', con=etl3_db_engine, schema=etl3_db_schema, if_exists="replace", index=False)
+
+
+    print("Update main tables with sirius IDs from map tables...")
```

Comment:
I think this might be cool as a map and a function to reuse code for updating all these or it might be overkill. It depends if there's gonna be lots of other tables to update with the full data set. 

Just like some way where you can kinda see whats being updated in one place. Again.. really depends on how hard that is to do and whether we're going to have shed loads of these or not.